openstack/limes/templates/_utils.tpl

```diff
@@ -20,7 +20,10 @@
       name: limes-secret
       key: rabbitmq_password
 - name: LIMES_AUDIT_RABBITMQ_USERNAME
-  value: "rabbitmq"
+  valueFrom:
+    secretKeyRef:
```

Comment:
I don't think we need to wrap the username in a secret.

Reply:
Sure, it is not necessary, but more consistent. I've changed it though to a "direct" value.